addons/delivery/views/payment_provider_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="payment_provider_custom_form" model="ir.ui.view">
```

Comment:
```suggestion
    <record id="payment_provider_form" model="ir.ui.view">
```
When you're ref it, it'll look like `delivery.payment_provider_form`, which has all the information we need.